src/main/java/mate/academy/dbconnection/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.dbconnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
The DB_URL is incomplete. It should contain the full JDBC URL to the database, including the host, port, and database name. For example: jdbc:mysql://localhost:3306/myDatabase